components/cli/cmd/cellery/setup_create_gcp.go

```diff
@@ -19,20 +19,51 @@
 package main
 
 import (
-	"github.com/spf13/cobra"
-
+	"fmt"
 	"github.com/cellery-io/sdk/components/cli/pkg/commands"
+	"github.com/spf13/cobra"
 )
 
 func newSetupCreateGcpCommand() *cobra.Command {
+	var isOutput bool
+	var addGlobalGW bool
+	var addObservability bool
 	cmd := &cobra.Command{
 		Use:   "gcp",
 		Short: "Create a Cellery runtime in gcp",
 		Args:  cobra.NoArgs,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if addObservability && !addGlobalGW {
+				return fmt.Errorf("You can't deploy observability portal without the Global gateway")
+			}
+			return nil;
+		},
 		Run: func(cmd *cobra.Command, args []string) {
-			commands.RunSetupCreateGcp()
+			if !addGlobalGW && !addObservability {
+				err := commands.CreateMinimalGcpRuntime()
+				if err != nil {
+					fmt.Printf("cellery : %v:\n", err)
```

Comment:
Shall we use `fmt.Errorf()` in here and other places where we log errors?

Reply:
We have to use log API for that. we will be doing that in a separate PR